### ISSUE TITLE
EditableColumn seems to ignore editableOptions

### ISSUE THREAD

**Issue by philippfrenzel** [kartik-v/yii2-grid#206]
Not working:(

```
[
                'class' => '\kartik\grid\EditableColumn',                
                'editableOptions' => function ($model, $key, $index) {                    
                    return [                        
                        'header'=>'Versicherer', 
                        'size'=>'md',
                        'inputType' => Editable::INPUT_TYPEAHEAD,
                        'options'=>[
                            'dataset' => [
                                [
                                    'remote' => Url::to(['/polizzen/versicherer-list']) . '?q=%QUERY',
                                    'limit' => 10
                                ]
                            ]
                        ],
                        'pluginOptions' => [
                            'highlight'=>true,                            
                        ],                        
                        'afterInput'=>function ($form, $widget) use ($model, $index) {
                            echo Html::hiddenInput('id', $model->epicid);
                            echo Html::hiddenInput('index',$index);
                            echo Html::hiddenInput('polizzen', 'versicherer');
                            echo Html::hiddenInput('polizzen_id',$model->id);                            
                        }
                    ];
                },
                'attribute'=>'versicherer', 
                'hAlign'=>'right', 
                'vAlign'=>'middle',
                'width'=>'150px',
                'pageSummary' => true,
            ],
```

Thanks Philipp

**Comment by philippfrenzel** [comment]
i removed this check:

&& $this->editableOptions instanceof Closure

and then it worked out fine! So how can I make the parameters an Instance of closure?

**Comment by kartik-v** [comment]
A callback like above is already an instanceof Closure. You may need to debug what's happening.

**Comment by kartik-v** [comment]
Resolved via #207.
